objcgen/embedder.cs

```diff
@@ -629,7 +642,8 @@ public int Compile ()
 								mmp.Append ("--debug ");
 							mmp.Append ("-p "); // generate a plist
 							mmp.Append ($"--target-framework {GetTargetFramework ()} ");
-							mmp.Append ($"\"--link_flags=-force_load {Path.GetFullPath (sdk_output_file)}\" ");
+							string extensionFlag = Extension ? "-fapplication-extension" : "";
```

Comment:
Why is this needed? It looks unrelated.

Reply:
looks like my other pull request has got merged into this pull

Reply:
You can just add another commit that reverts these bits and we'll squash it away on land.